frontend-react/src/components/FileHandlers/FileHandlerForm.tsx

```diff
@@ -35,7 +41,12 @@ export const FileHandlerForm = ({
     formLabel,
     resetText,
     submitText,
+    schemaOptions,
+    selectedSchemaOption,
+    onSchemaChange,
 }: FileHandlerFormProps) => {
+    const isDisabled = fileName.length === 0 || !selectedSchemaOption;
+
     return (
         <Form onSubmit={(e) => handleSubmit(e)} className="rs-full-width-form">
             {!submitted && (
```

Comment:
Not sure if this is outside the scope of this PR, but it seems like we're avoiding usage of `<FileInput />` usage of the `accept` prop that you can [see in practice in their Storybook](https://trussworks.github.io/react-uswds/?path=/story/components-file-input--accept-text-and-pdf).

This would help us avoid a lot of extra logic of determining selection AFTER the fact. It's quite weird that I can upload a file that is NOT supported (a `.pdf`) and it persists in the UI.

Reply:
Sure, can definitely add this!  I do want to note that since this is up to the browsers to interpret MIME types, there's a possibility of lesser-used files like HL7 (and maybe other formats in the future) not being respected by this attribute.  I don't think it'd hurt to add it in now so it's something we can just keep an eye out for in the future!